clambcc/clambc-compiler.py

```diff
@@ -444,16 +530,27 @@ def getOutputString(linked: IRFile, ignore: IRFile) -> str:
 def createOptimizedTmpFile(clangLLVM: ClangLLVM, linkedFile: str) -> str:
     name = getOptimizedTmpFileName(linkedFile)
 
-    cmd = f"{clangLLVM.getOpt()} \
+    cmd = f'{clangLLVM.getOpt()} \
             -S \
             {linkedFile} \
             -o {name} \
-            -internalize -internalize-public-api-list=entrypoint \
-            -globalopt \
-            "
-
-    ret = run(cmd)
+            -internalize-public-api-list=entrypoint \
+            --passes="internalize,globalopt" \
+            '
```

Comment:
dead code: Looks like you overwrote `cmd` after this.

Reply:
done